starters/vue-starter-ts/components/Pagination.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import ArrowLeft from './icons/ArrowLeft.vue';
+import ArrowRight from './icons/ArrowRight.vue';
+
+const props = defineProps<{
+  totalCount: number,
+  pageSize: number,
+  currentPage: number,
+  onChange: (page: number) => void,
+  disabled: boolean
+}>();
+
+const pageCount = computed(() => Math.ceil(props.totalCount / props.pageSize));
+const showPrevButton = computed(() => props.currentPage > 0);
+const showNextButton = computed(() => props.currentPage + 1 < pageCount.value);
+
+const goToNextPage = () => {
+  const newPage = Math.min(props.currentPage + 1, pageCount.value - 1);
+  props.onChange(newPage);
+};
+
+const goToPreviousPage = () => {
+  const newPage = Math.max(props.currentPage - 1, 0);
+  props.onChange(newPage);
+};
+</script>
+
+<template>
+  <div class="my-5 flex items-center">
```

Comment:
This needs an extra `justify-center` to get them centered horizontally like in the other starters